src/tests/ftest/util/test_utils_pool.py

```diff
@@ -603,7 +604,7 @@ def exclude(self, ranks, tgt_idx=None):
             CmdResult: Object that contains exit status, stdout, and other information.
 
         """
-        return self.dmg.pool_exclude(self.identifier, ranks, tgt_idx)
+        return self.dmg.pool_exclude(self.identifier, ranks, tgt_idx, force=True)
```

Comment:
Can we pass this argument through the function please? I.e.
`def exclude(self, ranks, tgt_idx=None, force=True):`
Not that we need to handle on this PR, but ideally tests should only use force if needed, so adding the argument means new tests can choose